schema_salad/makedoc.py

```diff
@@ -227,7 +234,7 @@ def typefmt(self,
                 nbsp=False,             # type: bool
                 jsonldPredicate=None    # type: Optional[Dict[str, str]]
                 ):
-        # type: (...) -> Union[str, unicode]
+        # type: (...) -> Union[str, Text]
```

Comment:
@manu-chroma It seems, though, that simple Text is enough.
"When checking Python 2.7 code, type checkers should treat the int and long types as equivalent. For parameters typed as unicode or Text , arguments of type str should be acceptable." from [PEP484](https://www.python.org/dev/peps/pep-0484/#stub-files)
I checked this with mypy, str and unicode pass for Text in python2 and str for Text in python3